sdmetrics/reports/multi_table/quality_report.py

```diff
@@ -25,6 +27,26 @@ def __init__(self):
         self._properties_scores = {}
         self._is_generated = False
         self._package_version = None
+        self._property_errors = {}
+
+    def _print_results(self, out=sys.stdout):
+        """Print the quality report results."""
+        if pd.isna(self._overall_quality_score) & any(self._property_errors.values()):
+            out.write('\nOverall Quality Score: Error computing report.\n\n')
+        else:
+            out.write(
+                f'\nOverall Quality Score: {round(self._overall_quality_score * 100, 2)}%\n\n')
+
+        if len(self._properties_instances) > 0:
+            out.write('Properties:\n')
+
+        for property_name, score in self._properties_scores.items():
+            if not pd.isna(score):
+                out.write(f'{property_name}: {round(score * 100, 2)}%\n')
+            elif property_name in self._property_errors:
```

Comment:
Remove this, so we always just return it as `nan` (or remove the check below, so we always output an error when the score is `nan`). This let's me remove the `_property_errors` attribute and simplify the `generate` code a bit.

Reply:
I think you can get nan without it being an error